### ISSUE TITLE
Handling missing subcommands

### ISSUE THREAD

**Issue by agrover** [stratis-storage/stratis-cli#43]
perhaps default to help text instead?

```
Traceback (most recent call last):
  File "./bin/stratis", line 39, in <module>
    main()
  File "./bin/stratis", line 36, in main
    return next(execution)
  File "/home/agrover/git/stratis-cli/src/stratis_cli/_main.py", line 28, in run
    args.func(args)
AttributeError: 'Namespace' object has no attribute 'func'
```

**Comment by agrover** [comment]
Should we have `stratis` be equivalent to `stratis pool`?

This is also true of subcommands with (e.g. `stratis pool`. Should we have these be equivalent to e.g. `stratis pool list`?

(If yes to both, then it follows that `stratis` would list pools.)

**Comment by mulkieran** [comment]
The first is fixed in PR #42. I'm not sure what you're asking in the second.

**Comment by mulkieran** [comment]
I think this is taken care of. Please reopen if it is not.
